homeassistant/components/overseerr/const.py

```diff
@@ -16,13 +17,32 @@
 
 EVENT_KEY = f"{DOMAIN}_event"
 
+MEDIA_EVENT_TYPES = {
+    "pending": "pending",
+    "approved": "approved",
+    "available": "available",
+    "failed": "failed",
+    "declined": "declined",
+    "auto_approved": "auto_approved",
+}
+ISSUE_EVENT_TYPES = {
+    "reported": "reported",
+    "commented": "comment",  # The notification type is ISSUE_COMMENT, but it looks better as "commented"
```

Comment:
so if we want the state to be commented, why isnt the strings.json updated to reflect that?